### PR TITLE
Assignment redone

### PR DESCRIPTION
**Pull request by NataliMarkova**
I completed the assignment 2 weeks ago and it was approved. After that, I decided to improve my code and add for loop to create a game board.
@marymatt504, the code with fool loop works now. Thank you for your help and useful link.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,8 +1,87 @@
...
+  };
+
+  // asking for the players' name
+  playerOne.name = prompt("Player 1 what is your name?");
+  playerTwo.name = prompt("Player 2 what is your name?");
+  
+  // randomly add ships to each board
+  const shipPlace = (player) => {
+    let x = Math.floor(Math.random() *4)
+      let y = Math.floor(Math.random() *4)
```
Comment (andresochoa91) [inline index.js:38]: Weird indentation.
Missing semicolons.

File: index.js [hunk 2]
```diff
@@ -1,8 +1,87 @@
...
+      } 
+    }
+
+  while(playerOne.shipCount < 4) {
+    shipPlace(playerOne);
+  }
+
+  while(playerTwo.shipCount < 4) {
+    shipPlace(playerTwo);
+  }
```
Comment (andresochoa91) [inline index.js:53]: This while loops are doing the same thing. We could create a functions and pass `playerOne` and `playerTwo` as arguments (not mandatory, personal preference).

**Review by andresochoa91 — APPROVED** [review]
This code is fantastic, clean and easy to read, great job!. Some suggestions right below.

**Review by andresochoa91 — APPROVED** [review]

File: index.js (hunk 1 above)
Comment (NataliMarkova) [reply index.js:38]: @andresochoa91 you right, didn't notice it.

File: index.js (hunk 2 above)
Comment (NataliMarkova) [reply index.js:53]: @andresochoa91, will try it. Thank you!

**Comment by NataliMarkova** [comment]
@andresochoa91 Once you approved my pull request I'll merge the code in master

**Pull request merged**